packages/react/src/primitives/SelectField/__tests__/SelectField.test.tsx

```diff
@@ -286,5 +286,24 @@ describe('SelectField test suite', () => {
         expect(option.innerHTML).toBe(optionString);
       });
     });
+
+    it('logs a warning to the console if the customer passes both children and options', async () => {
+      const warningMessage =
+        'Amplify UI: <SelectField> component  defaults to rendering children over `options`. When using the `options` prop, omit children.';
+      const spy = jest.spyOn(console, 'warn');
```

Comment:
These warnings will still show in the console when running the unit tests. Can we temporarily overwrite the console.warn function for your test, and then add the original back?
```
const originalWarn = console.warn;
console.warn = jest.fn();
// do your test stuff
console.warn = originalWarn;
```